indico/modules/rb/client/js/util.jsx

```diff
@@ -234,6 +237,67 @@ export function serializeRecurrenceInfo({type, number, interval}) {
   }
 }
 
+/**
+ * Renders the array of recurrence weekdays into a nicely formatted string
+ * @param {array} weekdays Array of weekdays
+ * @returns {string} Formatted string of weekdays
+ */
+export function renderRecurrenceWeekdays(weekdays) {
+  const order = ['mon', 'tue', 'wed', 'thu', 'fri', 'sat', 'sun'];
+  const weekdaysMap = {
+    mon: 'Monday',
+    tue: 'Tuesday',
+    wed: 'Wednesday',
+    thu: 'Thursday',
+    fri: 'Friday',
+    sat: 'Saturday',
+    sun: 'Sunday',
+  };
+
+  if (!weekdays || weekdays.length === 0) {
```

Comment:
```suggestion
  if (weekdays.length === 0) {
```

unless you have cases where we really want to pass `null` or similar